src/views/name.njk

```diff
@@ -2,7 +2,13 @@
 
 {% block content %}
 
-{% set partnershipType = props.parameters['which-type'] %}
+{% if props.data.cache['which-type'] %}
+  {% set partnershipType = props.data.cache['which-type'] %}
+{% elif props.data.limitedPartnership.partnership_type %}
+  {% set partnershipType = props.data.limitedPartnership.partnership_type %}
```

Comment:
Will the code ever fall in this block? i.e. do we remove the value in the (Redis) cache after we've successfully saved the value to Mongo?

Reply:
Value removed from cache after saving to db